src/controller/SetUpCodePairer.cpp

```diff
@@ -29,12 +29,17 @@
 #include <controller/CHIPDeviceController.h>
 #include <lib/dnssd/Resolver.h>
 #include <lib/support/CodeUtils.h>
+#include <system/SystemClock.h>
+
+constexpr uint32_t kDeviceDiscoveredTimeout = 30 * chip::kMillisecondsPerSecond;
```

Comment:
This policy choice/heuristics now becomes a policy shared by all. What is the justification for this particular non-configurable time-out value?

Reply:
@vivien-apple @raju-apple ?

Reply:
I have updated the code so there is a `#define` in `ChipConfig.h` instead of an hardcoded value.